application/classes/Ushahidi/Validator/Post/Decimal.php

```diff
@@ -13,7 +13,7 @@ class Ushahidi_Validator_Post_Decimal extends Ushahidi_Validator_Post_ValueValid
 {
 	protected function validate($value)
 	{
-		if (!Valid::decimal($value)) {
+		if (!Valid::numeric($value)) {
```

Comment:
@rjmackay I'm not getting an error when I enter 1, it should tell  me that I have to enter a decimal value but it's just passing, is that desired behaviour now, that a decimal can also not have a number of decimal places?

The field is also no longer displaying on the post detail.